### PR TITLE
Improve ensure-specs-running

### LINKED ISSUE #75: post-deploy check for kube-system-specs overly restrictive
The post-deploy script in question checks that there are exactly 6 running pods in the `kube-system` namespace.  It's natural to want to add other things to that namespace, such as the [`registry` addon](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/registry), which is required for using things like [Azure draft](https://github.com/Azure/draft/blob/master/docs/install.md#enable-minikube-add-ons).

### PR DESCRIPTION
**Pull request by Amit-PivotalLabs**
This rewrites `ensure-specs-running` in Go and improves
the logic, checking more intentionally for the desired state
within the `kube-system` namespace. In particular, this 
allows for other deployments to be defined in the `kube-system`
namespace.

[fixes #75]

### CODE REVIEW

File: src/ensure-specs-running/main.go [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    if _, hasPodTemplateHash := pod.ObjectMeta.Labels["pod-template-hash"]; !hasPodTemplateHash {
+        return unrecognized
+    }
+
+    numLabels := len(pod.ObjectMeta.Labels)
+    task := pod.ObjectMeta.Labels["task"]
+    k8sApp := pod.ObjectMeta.Labels["k8s-app"]
+    version := pod.ObjectMeta.Labels["version"]
+
+    if numLabels == 4 && task == "monitoring" && k8sApp == "heapster" && version == "v6" {
```
Comment (mkjelland) [inline src/ensure-specs-running/main.go]: Also it would be nice to use the constant strings you created above (on line 45) for "heapster", "influxdb", etc

**Comment by Amit-PivotalLabs** [comment]
Thanks for the feedback @mkjelland, PR updated.

**Comment by jhvhs** [comment]
Hi @Amit-PivotalLabs,

We have updated our release in order to allow additional specs to be added.
Please take a look at https://github.com/cloudfoundry-incubator/kubo-release/pull/95

**Comment by freynca** [comment]
Closing, per https://github.com/cloudfoundry-incubator/kubo-release/pull/95

**Pull request closed without merging**